packages/seacas/libraries/ioss/src/Ioss_Region.h

```diff
@@ -313,6 +314,10 @@ namespace Ioss {
     bool load_group_mesh(const std::string &child_group_name);
     bool load_group_mesh(const int child_group_index);
 
+    std::string get_group_name() const;
+
+    IOSS_NODISCARD std::tuple<std::string, int, double> locate_db_state(const double targetTime);
```

Comment:
Some compilers/analyzers will warn/complain about value parameters with `const` since it doesn't matter if it is const or not at this level since a copy is being passed and not the address of the clients value.

Reply:
Ok, I can change that programming style then ... I'm used to the idea of making sure that all possible arguments that *can* be const *should* be const